src/modal/modal-window.ts

```diff
@@ -1,20 +1,25 @@
 import {
+  AfterViewInit,
   Component,
-  Output,
+  ContentChild,
+  ContentChildren,
+  ElementRef,
   EventEmitter,
   Input,
-  ElementRef,
-  Renderer,
+  OnDestroy,
   OnInit,
-  AfterViewInit,
-  OnDestroy
+  Output,
+  Renderer
 } from '@angular/core';
 
 import {ModalDismissReasons} from './modal-dismiss-reasons';
 
+let nextId = 0;
+
 @Component({
   selector: 'ngb-modal-window',
   host: {
+    '[attr.aria-modal]': 'true',
```

Comment:
I don't see this attribute being added in the Bootstrap 4 documentation: https://v4-alpha.getbootstrap.com/components/modal/

Could you please open an issue in the Bootstrap repository first to clarify if this attribute is needed or not (when opening an issue please reference any specs / articles that made you add this attribute).